dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/config/StoreConfiguration.java

```diff
@@ -43,6 +44,7 @@ public StorageOperate storageOperate() {
         switch (PropertyUtils.getString(RESOURCE_STORAGE_TYPE)) {
             case STORAGE_S3:
                 return S3Utils.getInstance();
+            case STORAGE_LOCAL:
```

Comment:
Does we missing this statement?
```suggestion
            case STORAGE_LOCAL:
                return HadoopUtils.getInstance();
```